pro/src/pages/AdageIframe/app/components/OffersInstantSearch/OffersSearch/Offers/__spec__/Offer.spec.tsx

```diff
@@ -486,4 +493,40 @@ describe('offer', () => {
 
     expect(screen.queryByText('Enregistrer en favoris')).not.toBeInTheDocument()
   })
+
+  it('should log info when clicking "en savoir plus" button for no result page suggestion', async () => {
```

Comment:
(suggestion) En lisant juste le nom du test j'ai un peu de mal à comprendre ce qu'il fait, est-ce qu'on pourrait faire 2 tests différents "should log event when clicking..." et "should mention if the log is from a suggestion offer" ?